tools/scripts/seed/certifiedUserData.js

```diff
@@ -0,0 +1,4655 @@
+/* eslint-disable max-len */
+const ObjectId = require('mongodb').ObjectID;
+const defaulUserImage = require('../../../config/misc').defaulUserImage;
```

Comment:
```suggestion
const defaultUserImage = require('../../../config/misc').defaultUserImage;
```
This was fixed fairly recently.